crates/pallets/starknet/src/tests/mod.rs

```diff
@@ -11,3 +18,194 @@ mod sequencer_address;
+
+    InvokeTransaction {
+        version: 1,
+        sender_address,
+        calldata,
+        nonce,
+        signature,
+        max_fee: Felt252Wrapper::from(u128::MAX),
+    }
+    .from_invoke(Starknet::chain_id())
+}
+
+fn get_invoke_braavos_dummy() -> Transaction {
+    let signature = bounded_vec!(
+        Felt252Wrapper::from_hex_be("0x00f513fe663ffefb9ad30058bb2d2f7477022b149a0c02fb63072468d3406168").unwrap(),
+        Felt252Wrapper::from_hex_be("0x02e29e92544d31c03e89ecb2005941c88c28b4803a3647a7834afda12c77f096").unwrap(),
+    );
+    let sender_address =
+        Felt252Wrapper::from_hex_be("0x05ef3fba22df259bf84890945352df711bcc9a4e3b6858cb93e9c90d053cf122").unwrap();
+    let nonce = Felt252Wrapper::ZERO;
+    let calldata = bounded_vec!(
+        Felt252Wrapper::from_hex_be("0x0000000000000000000000000000000000000000000000000000000000000001").unwrap(),
+        Felt252Wrapper::from_hex_be("0x024d1e355f6b9d27a5a420c8f4b50cea9154a8e34ad30fc39d7c98d3c177d0d7").unwrap(),
+        Felt252Wrapper::from_hex_be("0x00e7def693d16806ca2a2f398d8de5951344663ba77f340ed7a958da731872fc").unwrap(),
+        Felt252Wrapper::from_hex_be("0x0000000000000000000000000000000000000000000000000000000000000000").unwrap(),
+        Felt252Wrapper::from_hex_be("0x0000000000000000000000000000000000000000000000000000000000000001").unwrap(),
+        Felt252Wrapper::from_hex_be("0x0000000000000000000000000000000000000000000000000000000000000001").unwrap(),
+        Felt252Wrapper::from_hex_be("0x0000000000000000000000000000000000000000000000000000000000000019").unwrap(),
+    );
+
+    InvokeTransaction {
+        version: 1,
+        sender_address,
+        calldata,
+        nonce,
+        signature,
+        max_fee: Felt252Wrapper::from(u128::MAX),
+    }
+    .from_invoke(Starknet::chain_id())
+}
+
+fn get_invoke_emit_event_dummy() -> Transaction {
+    let signature = bounded_vec!(
+        Felt252Wrapper::from_hex_be("0x00f513fe663ffefb9ad30058bb2d2f7477022b149a0c02fb63072468d3406168").unwrap(),
+        Felt252Wrapper::from_hex_be("0x02e29e92544d31c03e89ecb2005941c88c28b4803a3647a7834afda12c77f096").unwrap(),
+    );
+    let sender_address =
+        Felt252Wrapper::from_hex_be("01a3339ec92ac1061e3e0f8e704106286c642eaf302e94a582e5f95ef5e6b4d0").unwrap();
+    let nonce = Felt252Wrapper::ZERO;
+    let calldata = bounded_vec!(
+        Felt252Wrapper::from_hex_be("0x024d1e355f6b9d27a5a420c8f4b50cea9154a8e34ad30fc39d7c98d3c177d0d7").unwrap(),
+        Felt252Wrapper::from_hex_be("0x00966af5d72d3975f70858b044c77785d3710638bbcebbd33cc7001a91025588").unwrap(),
+        Felt252Wrapper::from_hex_be("0x0000000000000000000000000000000000000000000000000000000000000000").unwrap()
+    );
+
+    InvokeTransaction {
+        version: 1,
+        sender_address,
+        calldata,
+        nonce,
+        signature,
+        max_fee: Felt252Wrapper::from(u128::MAX),
+    }
+    .from_invoke(Starknet::chain_id())
+}
+
+fn get_invoke_nonce() -> Transaction {
```

Comment:
`get_invoke_nonce_dummy` won't block for this but if you push some more commits on the PR please update